lib/services/android-project-service.ts

```diff
@@ -8,9 +8,12 @@ import options = require("../common/options");
 import constants = require("../constants");
 import hostInfo = require("../common/host-info");
 import helpers = require("../common/helpers");
+import fs = require("fs");
```

Comment:
`fs` is still used for `fs.appendFileSync`. I checked `IFileSystem` and I does not offer method for appending. Any suggestions?